nss-day-cohort-43-csharp/Fullstack--span--Halen
# Tag list

# Description
When logged in as admin, you should be able to view a list of all tags.
Fixes # (issue)
## Type of change
Please delete options that are not relevant.
# Testing Instructions
Log in as a admin, view all tags. Log in as author and you should not be able to view tags.
# Checklist:
